packages/swatch/src/SwatchGroup.ts

```diff
@@ -50,11 +53,26 @@ export class SwatchGroup extends SizedMixin(SpectrumElement, {
     @property({ reflect: true })
     public border: SwatchBorder;
 
+    @property({ reflect: true })
+    public density: 'compact' | 'spacious' | undefined;
+
     @property({ reflect: true })
     public rounding: SwatchRounding;
 
     @property({ type: Array })
-    public selected: string[] = [];
+    public get selected(): string[] {
+        return this._selected;
+    }
+
+    public set selected(selected: string[]) {
+        if (selected === this.selected) return;
+
+        const oldSelected = this.selected;
+        this._selected = selected;
+        this.requestUpdate('selected', oldSelected);
+    }
+
+    private _selected: string[] = [];
```

Comment:
Since this looks identical to the generated accessors, it may be worth adding a comment explaining that it's to allow setting `_selected` internally without triggering setter behavior.